ninjato-core/src/main/kotlin/com/agoda/ninjato/misc/BodyExtensions.kt

```diff
@@ -7,10 +7,13 @@ import com.agoda.ninjato.http.MediaType
 import com.agoda.ninjato.http.Parameters
 import com.agoda.ninjato.http.Request.Configurator.WithBody
 import com.agoda.ninjato.reflect.TypeReference.Companion.reifiedType
+import java.io.File
 
 inline fun WithBody.formUrlEncoded(tail: Parameters.() -> Unit)
         = Body(Parameters().apply(tail).resolve().toUrlEncoded(), MediaType.FormUrlEncoded())
 
+fun WithBody.file(file: File, mediaType: MediaType) = Body(file.readBytes(), mediaType)
```

Comment:
Could you please make it inline function?

Reply:
done